tijazz/Property-Pro-Lite
# #166812454 Build Users landing page and Folder directory structure

What does this PR do?

Have the landing page (index.html) created for all users
Description of Task to be completed?

The navigation bar can be used to route users to various pages such as about.html, contact.html
How should this be manually tested?

After cloning the repo, click on index.html. This will display the page on chrome or other compatible browsers.
Any background context you want to provide?

null
What are the relevant pivotal tracker stories?

#166812454
For the full pivotal story, check the ID above
